install-dev/data/xml/hook.xml

```diff
@@ -965,112 +965,112 @@
       <name>actionSqlRequestFormBuilderModifier</name>
       <title>Modify sql request identifiable object form</title>
       <description>This hook allows to modify sql request identifiable object forms content by modifying form
-          builder data or FormBuilder itself
+      builder data or FormBuilder itself
```

Comment:
Why do you need an EOL ? Why not all in the same line ?

Reply:
automatic indentation :sweat:

Reply:
Do you fix or Do i valid ?